metaphor/openapi/config.py

```diff
@@ -20,13 +20,36 @@ class OpenAPIAuthConfig:
 
 
 @dataclass(config=ConnectorConfig)
-class OpenAPIRunConfig(BaseConfig):
+class OpenAPIJsonConfig:
     base_url: HttpUrl
     openapi_json_path: Optional[FilePath] = None
     openapi_json_url: Optional[HttpUrl] = None
     auth: Optional[OpenAPIAuthConfig] = None
 
     @model_validator(mode="after")
-    def have_path_or_url(self) -> "OpenAPIRunConfig":
+    def have_path_or_url(self) -> "OpenAPIJsonConfig":
         must_set_exactly_one(self.__dict__, ["openapi_json_path", "openapi_json_url"])
         return self
+
+
+@dataclass(config=ConnectorConfig)
+class OpenAPIRunConfig(BaseConfig):
+    base_url: Optional[HttpUrl] = None
+    openapi_json_path: Optional[FilePath] = None
+    openapi_json_url: Optional[HttpUrl] = None
+    auth: Optional[OpenAPIAuthConfig] = None
+    specs: Optional[List[OpenAPIJsonConfig]] = None
```

Comment:
```suggestion
    specs: List[OpenAPIJsonConfig]] = field(default_factory=list)
```
And then add a validator to ensure `specs` isn't empty?

Reply:
We try not to introduce a breaking change, we can use your suggestion when we bump to 0.15